lib/features/recurring_donations/create/widgets/create_recurring_donation_bottom_sheet.dart

```diff
@@ -19,27 +20,39 @@ import 'package:go_router/go_router.dart';
 import 'package:intl/intl.dart';
 
 class CreateRecurringDonationBottomSheet extends StatelessWidget {
-  const CreateRecurringDonationBottomSheet({super.key});
+  const CreateRecurringDonationBottomSheet({
+    super.key,
+    this.recipient,
+  });
+
+  final CollectGroup? recipient;
 
   @override
   Widget build(BuildContext context) {
+    final cubit = CreateRecurringDonationCubit(
```

Comment:
is build method the best place to provide cubit?

Reply:
I had to do it like this because we don't have a route for this and i am creating the cubit when we init the page